src/Forside/EkspanderbarSammenligning/EkspanderbarSammenligning.tsx

```diff
@@ -79,6 +79,7 @@ export const EkspanderbarSammenligning: FunctionComponent<Props> = ({
             <SlikHarViKommetFramTilDittResultat
                 resultat={sammenligningResultatTotalt.sammenligningVurdering}
                 kvartaler={sammenligningResultatTotalt.kvartaler}
+                //TODO finn ut hvor dette blir brukt i amplitude før vi endrer
                 onÅpne={() => sendEvent('barnehage sammenligning lesmer', 'åpne')}
```

Comment:
Ser ikke ut til at det brukes i noen grafer. Tror vi trygt kan fjerne dette Amplitude-kallet. 

![image](https://user-images.githubusercontent.com/17157469/133240372-ca0526aa-40fa-4ba8-a2f1-e74d6ad0e099.png)

Reply:
[Denne](https://analytics.amplitude.com/nav/chart/9z1zgf0?source=govern+details+panel) grafen er forøvrig relevant, selv om dette eventet ikke brukes der.